README.md

```diff
@@ -1,38 +1,89 @@
 # Encryptio 
-Generic encryption software
+Encrypt strings, decrypt files and determine encryption of encrypted files. 
 
 ## Prerequisites 
-
-## Usage
+install required python in version 3.11
+```bash
+apt install python3.11
+```
 
 install required libraries via requirements 
 ```bash
 pip install -r requirements.txt
 ```
 This will download and install the [cryptography](https://pypi.org/project/cryptography/) and [cryptodome](https://pypi.org/project/pycryptodome/#description) libraries. 
 
-encryptio uses a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+## Usage of encryptio.py 
+encryptio.py use a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+### encryption
+```bash
+python3 encryptio.py encryptio.py encryption [-h] {AES,DES,3DES,IDEA,BLOWFISH} cryptoKey {CBC,CFB,CTR,ECB,OFB} cryptoData
+```
+
+Where 
+* **{AES,DES,3DES,IDEA,BLOWFISH}** - algorithm to encrypt your string with
+* **cryptoKey** - key to encrypt your string with
+* **{CBC,CFB,CTR,ECB,OFB}** - mode of encryption
+* **cryptoData** - string to encrypt
 
+### decryption
 ```bash
-python3 encryptio.py [-h] encryptionAlgorithm encryptionKey encryptionLibrary encryptionObjectIdentifier (encryptionObject)
+python3 encryptio.py decryption [-h] {AES,DES,3DES,IDEA,BLOWFISH} cryptoKey {CBC,CFB,CTR,ECB,OFB} cryptoDataFile
 ```
 
 Where 
-* **encryptionAlgorithm** - the algorithm to encrypt your object with
-* **encryptionKey** - the string key to encrypt your object with
-* **encryptionLibrary** - the encryption library to encrypt with
-* **encryptionObjectIdentifier** - the identifier of the object to encrypt
-* **encryptionObject** - the options for the object to encrypt
+* **{AES,DES,3DES,IDEA,BLOWFISH}** - algorithm to decrypt your file with
+* **cryptoKey** - key to decrypt your file with
+* **{CBC,CFB,CTR,ECB,OFB}** - mode of encryption
+* **cryptoDataFile** - file to decrypt
 
-* **-o** - the optional output file to write the encrypted object to
+## Usage of determiner.py 
+determiner.py use a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
 
-### EncryptionObjects
+### determining
+```bash
+python3 determiner.py [-h] -f encryptedFile -k encryptionKey
```

Comment:
maybe you can reuse the same wording as above cryptoKey & cryptoDataFile or rename it above. would be prettier 🦩

Reply:
Done